### ISSUE TITLE
Route Voice to Device Speaker when Connected to Bluetooth

### ISSUE THREAD

**Issue by billyking991** [mapbox/mapbox-navigation-android#1933]
Is it possible to route the navigation voice to the device's speaker when the device connects to Bluetooth in an audio system?  If not, I would like to have that ability, if possible.  My app users have been requesting this ability for a little while.  

When I start my car, the only way I can hear the voice after my device connects to Bluetooth is if I set my radio input to the device.  If I'm listening to XM/FM, the voice won't play properly.  I would like the ability to play the voice over the device's speaker, if desired.   

Waze has a similar feature located in Settings -> Voice & Sound -> Play Sound Via.

**Android API:** - Android 5 - 9
**Mapbox Navigation SDK version:** - 0.37.0

### Steps to trigger behavior

 1.   Let device connect to Bluetooth
 2.  Set radio to XM/FM/AM
 3.  Start navigation

### Expected behavior
User should be able to choose where the voice is heard in the app Settings.  The choice should be between the car's audio or the device's speaker.

### Actual behavior
The voice automatically is routed to the car's audio system if the device is connected via Bluetooth.

**Comment by danesfeder** [comment]
Hey @billyking991 👋 are you referring to this option?  

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/8434572/57641500-24cb9c80-7583-11e9-847b-5376e1965954.gif)

**Comment by billyking991** [comment]
Yes!  I apologize...the path I had posted above was for iPhone.  I would love to be able to add that feature to my Mapbox app.  Is there any way to do that yet?

**Comment by billyking991** [comment]
I just had two more people asking for this feature.  If there isn't a way to route the voice programmatically, I would love the option to add it to my apps.

**Comment by cmahopper** [comment]
Joining in to say that we are also interested in this feature!

**Comment by billyking991** [comment]
Just checking back on this.  Is there a way to map the voice to the device speaker when the device is connected to the car audio with Bluetooth?  This is one of the most confusing things for my app users.  They can't hear the voice when the device is connected to Bluetooth, but they are listening to FM radio.  I've tried routing it with code, but have had no luck.

Right now, the only solutions are:

1.  Disconnect from Bluetooth.
2.  Set the radio input to Bluetooth which only allows them to listen to music played over the device itself.

Is there is a way to route the navigation voice over the device speaker when the device is connected to Bluetooth?

**Comment by strattas** [comment]
Same here please get this feature. Its very strage this day and age to not have this work,

**Comment by billyking991** [comment]
I've tried to figure this out on my end, but I have been unable to do so thus far.  If you know of a way I can reroute the voice using a bit of code, please let me know.  I will give it a try on my end.

**Comment by Guardiola31337** [comment]
@asinghal22 is this something we want to support in `1.0`?

cc @JunDai @abhishek1508
